cookbook/views/new.py

```diff
@@ -71,6 +70,31 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class ConnectorConfigCreate(GroupRequiredMixin, CreateView):
+    groups_required = ['admin']
+    template_name = "generic/new_template.html"
+    model = ConnectorConfig
+    form_class = ConnectorConfigForm
+    success_url = reverse_lazy('list_connector_config')
+
+    def form_valid(self, form):
+        if self.request.space.demo or settings.HOSTED:
```

Comment:
i guess now that you added that extra flag to disable this feature, lets tie it to that and say "not available in demo" for demo and "not enabled on this instance" so that I can enable it eventually